src/pages/VenuePage/VenuePage.tsx

```diff
@@ -74,7 +74,7 @@ const VenuePage = () => {
     user && venue && isUserAMember(user.email, venue.config?.memberEmails);
 
   const venueName = venue && venue.name;
-  useUpdateLocationEffect(user, venueName);
+  useUpdateLocationEffect(user, venueName); // Camp and PartyMap needs to be able to modify this
```

Comment:
Either need a way for child components to update the venue name, or put this back in individual venues. Not a big fan of the 2 way controls (`props` and child) but probably still better. Thoughts?

Reply:
can you provide venue as the argument and then determine what to send in the hook itself?